rubocop-minitest.gemspec

```
@@ -32,6 +32,6 @@ Gem::Specification.new do |spec|
   spec.executables = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_runtime_dependency 'rubocop', '>= 0.90', '< 2.0'
+  spec.add_runtime_dependency 'rubocop', '>= 1.43', '< 2.0'
```

Upgraded rubocop because I need `range_with_comments_and_lines` from `RangeHelp` module. 
Probably better done as a separate PR?

---

It's a big jump, but let's still include it in this PR. By the way, it looks like this was introduced in 1.39, not 1.43.
https://github.com/rubocop/rubocop/commit/0d6b0d126f5ce887b496bd4ab801e53fad0c8956

```suggestion
  spec.add_runtime_dependency 'rubocop', '>= 1.39', '< 2.0'
```